ml3d/vis/visualizer.py

```diff
@@ -305,7 +305,7 @@ def _calc_pointcloud_size(self, raw_data, pcloud):
         for (attr, arr) in raw_data.items():
             pcloud_size += arr.size * 4
         # Point cloud consumes 64 bytes of per point of GPU memory
-        pcloud_size += pcloud.point["points"].size * 64
+        pcloud_size += pcloud.point["points"].num_elements() * 64
```

Comment:
`pcloud_size += pcloud.point["points"].num_elements() * pcloud.point["points"].dtype.byte_size()`

Reply:
The size calculation is supposed to reflect GPU memory, which includes other pieces of data (color, maybe normal), so dtype size would be too small.

Reply:
Sounds good. Can we get the right value without hard coding 64? We may want to use a different data type (eg: uint8 instead of float for color) that may throw this off.